src/components/PassphraseSetterBox.js

```diff
@@ -43,9 +44,10 @@ class PassphraseSetterBox extends Nimiq.Observable {
         /* eslint-disable max-len */
         $el.innerHTML = `
             <div class="password-strength strength-0  nq-text-s" data-i18n="passphrasebox-password-strength-0" >Enter at least 8 characters</div>
-            <div class="password-strength strength-8  nq-text-s" data-i18n="passphrasebox-password-strength-8" >Great, that's a good password!</div>
-            <div class="password-strength strength-10 nq-text-s" data-i18n="passphrasebox-password-strength-10">Super, that's a strong password!</div>
-            <div class="password-strength strength-12 nq-text-s" data-i18n="passphrasebox-password-strength-12">Excellent, that's a very strong password!</div>
+            <div class="password-strength strength-4  nq-text-s" data-i18n="passphrasebox-password-strength-4" >Hm, that's a weak password :(</div>
```

Comment:
`hm`, `:(`? 
```suggestion
            <div class="password-strength strength-4  nq-text-s" data-i18n="passphrasebox-password-strength-4" >That is a weak password!</div>
```
I suggest using `that is` over `that's`. I also suggest using punctuation evenly (including `strength-0`).

Reply:
I now adapted to your wording suggestions, however we will have to go over all wording together with Julian and Tammo anyway again.